Fix another flaky test

https://circleci.com/gh/neuromation/platform-api/5786

```
_ TestPodContainerDevShmSettings.test_shm_extended_not_requested_try_create_huge_file _

self = <tests.integration.test_kube_orchestrator.TestPodContainerDevShmSettings object at 0x7f30e419d948>
run_command_get_status = <function TestPodContainerDevShmSettings.run_command_get_status.<locals>._f at 0x7f30e5f24618>

    @pytest.mark.asyncio
    async def test_shm_extended_not_requested_try_create_huge_file(
        self, run_command_get_status: Callable[..., Awaitable[JobStatusItem]]
    ) -> None:
        command = "dd if=/dev/zero of=/dev/zero  bs=999999M  count=1"
        resources = ContainerResources(cpu=0.1, memory_mb=128, shm=False)
        status_actual = await run_command_get_status(resources, command)
        status_expected = JobStatusItem.create(
            status=JobStatus.FAILED,
            reason=JobStatusReason.OOM_KILLED,
            exit_code=137,
            description=None,
        )
>       assert status_actual == status_expected, f"actual: '{status_actual}'"
E       AssertionError: actual: 'JobStatusItem(status=JobStatus.FAILED, transition_time=datetime.datetime(2019, 11, 26, 10, 57, 37, 418878, tzinfo=datetime.timezone.utc), reason='OOMKilled', description='0+1 records in\n0+1 records out\n', exit_code=137)'
E       assert JobStatusItem...exit_code=137) == JobStatusItem...exit_code=137)
E         Matching attributes:
E         ['status', 'reason', 'exit_code']
E         Differing attributes:
E         description: '0+1 records in\n0+1 records out\n' != None

tests/integration/test_kube_orchestrator.py:1658: AssertionError
```

--- a/tests/integration/test_kube_orchestrator.py
+++ b/tests/integration/test_kube_orchestrator.py
@@ -5,6 +5,7 @@ import uuid
 from dataclasses import replace
 from pathlib import PurePath
 from typing import Any, AsyncIterator, Awaitable, Callable, Dict, Iterator, Optional
+from unittest import mock
 
 import aiohttp
 import pytest
@@ -1653,7 +1654,7 @@ class TestPodContainerDevShmSettings:
             status=JobStatus.FAILED,
             reason=JobStatusReason.OOM_KILLED,
             exit_code=137,
-            description=None,
+            description=mock.ANY,
         )
         assert status_actual == status_expected, f"actual: '{status_actual}'"
 

